OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -277,7 +277,7 @@
     <Compile Include="Traits\Attack\AttackOmni.cs" />
     <Compile Include="Traits\Attack\AttackSuicides.cs" />
     <Compile Include="Traits\Attack\AttackTurreted.cs" />
-    <Compile Include="Traits\Attack\AttackWander.cs" />
+    <Compile Include="Traits\AttackWander.cs" />
```

Comment:
Because the trait name is misleading, this is not an `AttackBase` derivative and not strictly an attack trait by itself, either.